### PR TITLE
Request for the formalization of the PUG token within Carbonswap Community List

### PR DESCRIPTION
**Pull request by pugafran**
Token that was conceived as a memetoken but has evolved over time as a utility token, PUG provides through our DEX (Decentralised Extension) an extra layer to carbonswap, through CLPs and tokens native to the PUG ecosystem you can enter farming pools, staking, lotteries, and more things that are being worked on such as NFTs.

We are the mafia that protects the energy chain from scammers and weak hands. 5% of the pugs that are sent in any kind of transaction are burned, or die, however you want to put it, the streets are dangerous, pugs are shot on every transaction and unfortunately the casualties are 5%...We want to be the biggest friendly mafia in ewc and all blockchains, true to clean energy, defending all susus patrolling carbonswap.

We have clean energy and helping as a fundamental pillar within the project, as will be seen.

Logo 250 x 250:
0x59b6196e41c118dfF75961257b882e86b915a0e8

### CODE REVIEW

File: carbonswap/CS_Community_List.json [hunk 1]
```diff
@@ -106,6 +106,19 @@
             "tags": [
                 "bridged"
             ]
+        },
+
```
Comment (TopSootPrime) [inline carbonswap/CS_Community_List.json:110]:
```suggestion
```

Comment (TopSootPrime) [reply carbonswap/CS_Community_List.json:110]: no useless empty line

File: carbonswap/CS_Community_List.json [hunk 2]
```diff
@@ -106,6 +106,19 @@
             "tags": [
                 "bridged"
             ]
+        },
+        
+        {
+            "chainId": 246,
+            "address": "0x59b6196e41c118dff75961257b882e86b915a0e8",
```
Comment (TopSootPrime) [inline carbonswap/CS_Community_List.json:113]:
```suggestion
            "address": "0x59b6196e41c118dfF75961257b882e86b915a0e8",
```

Comment (TopSootPrime) [reply carbonswap/CS_Community_List.json:113]: address needs to be checksummed

File: carbonswap/CS_Community_List.json [hunk 3]
```diff
@@ -106,6 +106,19 @@
...
+            "name": "Gangster Pug",
+            "decimals": 18,
+            "symbol": "PUG",
+            "logoURI": "https://raw.githubusercontent.com/carbonswap/assets/master/carbonswap/logo/0x59b6196e41c118dff75961257b882e86b915a0e8.png",
```
Comment (TopSootPrime) [inline carbonswap/CS_Community_List.json:117]:
```suggestion
            "logoURI": "https://raw.githubusercontent.com/carbonswap/assets/master/carbonswap/logo/0x59b6196e41c118dfF75961257b882e86b915a0e8.png",
```

Comment (TopSootPrime) [reply carbonswap/CS_Community_List.json:117]: address needs to be checksummed.
Can you add the token logo too as `0x59b6196e41c118dfF75961257b882e86b915a0e8.png` into `carbonswap/logo`?

**Review by TopSootPrime — CHANGES_REQUESTED** [review]

File: carbonswap/CS_Community_List.json (hunk 3 above)
Comment (pugafran) [reply carbonswap/CS_Community_List.json]: Fixed!

**Review by TopSootPrime — APPROVED** [review]
LGTM

**Pull request merged**